portfolio/src/main/webapp/script.js

```diff
@@ -54,11 +83,12 @@ class Portfolio {
         let res = await fetch(path);
 
         /* Check for errors in the HTTP response and alert the user. */
-        if (res.status == 500) {
-            alert('Error: Cannot display ' + this.numComments + 
-            ' comments because fewer than ' + this.numComments + ' comments exist.');
-        }
-        else if (res.status != 200) {
+        if (res.status === 404){
+            alert("Empty datastore.");
+        } else if (res.status === 500) {
+            alert('Error: invalid count requested. Fewer than ' + this.numComments + ' comments exist.');
```

Comment:
You can also use a template string: `` `... Fewer than ${this.numComments} comments exist.` ``